include/openssl/ec.h

```diff
@@ -860,6 +860,7 @@ int ECPKParameters_print_fp(FILE *fp, const EC_GROUP *x, int off);
 #  define EC_FLAG_NON_FIPS_ALLOW  0x1
 #  define EC_FLAG_FIPS_CHECKED    0x2
 #  define EC_FLAG_COFACTOR_ECDH   0x1000
+#  define EC_FLAG_SM2_RANGE       0x4
```

Comment:
I am not sure if this value is right or not...

Reply:
I would like to have an explicit answer for this question since the `EC_FLAG_COFACTOR_ECDH` looks strange.

Reply:
This is OK.